addons/mail/static/src/model/model_manager.js

```diff
@@ -687,7 +687,13 @@ export class ModelManager {
          */
         const recordMethods = registry.get(model.name).get('recordMethods');
         for (const methodName of recordMethods.keys()) {
-            record[methodName] = record[methodName].bind(record);
+            const boundedMethod = record[methodName].bind(record);
+            record[methodName] = (...args) => {
+                if (!record.exists()) {
+                    return;
+                }
+                return boundedMethod(...args);
+            };
```

Comment:
This should be done at model level instead of at create of every record. The existing `bind` can only be done per record, but this code is the same for all records.

Reply:
Done